src/components/ReportActionItem/TaskPreview.js

```diff
@@ -59,7 +65,8 @@ function TaskPreview(props) {
         ? props.taskReport.stateNum === CONST.REPORT.STATE_NUM.SUBMITTED && props.taskReport.statusNum === CONST.REPORT.STATUS.APPROVED
         : props.action.childStateNum === CONST.REPORT.STATE_NUM.SUBMITTED && props.action.childStatusNum === CONST.REPORT.STATUS.APPROVED;
     const taskTitle = props.taskReport.reportName || props.action.childReportName;
-    const taskAssignee = props.taskReport.managerEmail || props.action.childManagerEmail;
+    const taskAssigneeAccountID = TaskUtils.getTaskAssigneeAccountID(props.taskReport);
```

Comment:
We didn't consider the case where the `taskReport` could be empty and hence it caused a regression [here](https://github.com/Expensify/App/issues/22373), because the task assignment didn't show an email address.

Reply:
hmm if task report could be empty, would this have crashed before too? 🤔 (since we used to directly access `props.taskReport.managerEmail`